backend/routes/indexer.go

```diff
@@ -413,6 +416,26 @@ func processNewDayEvent(event IndexerEvent, w http.ResponseWriter) {
 	}
 }
 
+func processNFTTransferEvent(event IndexerEvent, w http.ResponseWriter) {
+	to := event.Event.Keys[2][2:] // Remove 0x prefix
+	tokenIdHex := event.Event.Keys[3]
```

Comment:
tokenid will be u256 so it will take 2 slots keys[3] and [4]. You can set it up to just use the low value for now, and leave a TODO in the code like here : https://github.com/keep-starknet-strange/art-peace/blob/25d773412f343fb04092d061282e819c30149b51/backend/routes/indexer/nft.go#L18